notebooks/QuickStart.ipynb

```diff
@@ -90,12 +90,12 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Enter JSON string: "
+      "Enter encoded string: "
```

Comment:
```suggestion
      "Enter ApertureDB Key "
```

Reply:
OK.  Obviously we should match the Python, but consistent naming there would be nice.

Reply:
I agree. we should change it there, then here.